src/pyodide/internal/sphinx/README.md

```diff
@@ -0,0 +1,15 @@
+## Python SDK docs markdown generator
+
+This directory contains config files for Sphinx, which can generate docs based on the exported methods in asgi.py and workers.py. These files are a part of the Python SDK.
+
+### Usage
+
+
+```
+# set up venv with sphinx and sphinx-markdown-builder installed
```

Comment:
Could you add a `requirements-doc.txt` so that this can say:
```
python -m venv .venv
source .venv/bin/activate
pip install -r requirements-doc.txt
make markdown
```